src/Map.js

```diff
@@ -538,6 +538,60 @@ toggleHandlers["showDwellLocations"] = function (enabled) {
   }
 };
 
+/*
+ * Draws circles on the map. Size indicates dwell time at that
+ * location.
+ */
+toggleHandlers["showNavStatus"] = GenerateBubbles(
+  "showNavStatus",
+  (rawLocationLatLng, lastLocation, le) => {
+    const navStatus = _.get(le, "jsonPayload.response.navStatus");
+    if (navStatus === undefined) {
+      return;
+    }
+    let color,
+      radius = 5;
+    switch (navStatus) {
+      case "NAVIGATION_STATUS_UNKNOWN_NAVIGATION_STATUS":
+        color = "#222222";
+        break;
+      case "NAVIGATION_STATUS_NO_GUIDANCE":
+        color = "#090909";
+        break;
+      case "NAVIGATION_STATUS_ENROUTE_TO_DESTINATION":
+        color = "#00FF00";
+        break;
+      case "NAVIGATION_STATUS_OFF_ROUTE":
+        color = "#FF0000";
+        radius = 30;
+        break;
+      case "NAVIGATION_STATUS_ARRIVED_AT_DESTINATION":
+        color = "0000FF";
+        radius = 10;
+        break;
+      default:
+        color = "#000000";
+    }
+    const statusCirc = new google.maps.Circle({
+      strokeColor: color,
+      strokeOpacity: 0.5,
+      fillColor: color,
+      fillOpacity: 0.5,
+      map,
+      center: rawLocationLatLng,
+      radius: radius, // set based on trip status?
+    });
+    google.maps.event.addListener(statusCirc, "mouseover", () => {
+      setFeaturedObject({
+        navStatus: navStatus,
+        vehicleState: _.get(le, "jsonPayload.response.state"),
+        tripStatus: "??",
```

Comment:
Can't find any reference in this patch to this key/value.  Just confirming this is intentional?

Reply:
Oh this value is defined/used in #96.  Does this code need to worry about overwriting?

Reply:
The keys are used purely for display (in the json viewer).    As you mouse over these circles, the contents of teh json viewer changes -- they're basically to provide context/raw data used by the visualization.

The values ... especially for this one 'jsonPayload.response.state' is from the logs.